apps/dashboard/app/views/system_status/index.turbo_stream.erb

```diff
@@ -0,0 +1,33 @@
+<turbo-stream action="replace" target="system-status">
+  <template>
+    <% @job_clusters.each do |c| %>
+      <div class="col-sm-6 col-xs-12 p-2">
+        <div class="col border rounded p-2 text-center">
+          <h4><strong><%= title c %></strong></h4>
```

Comment:
Be careful with headings. Semantically, they only kinda make sense if there are also h1-h3's on the page. an h4 by itself is going to throw screen reader users off if they don't see others.

If it's a sizing thing you can directly use the `h4` bootstrap class to get the appropriate text size.